server/routes/nl_interface.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Data Commons NL Interface routes"""
+
+import os
+
+from flask import current_app
+import flask
+from flask import Blueprint, render_template
+
+import services.datacommons as dc
+
+bp = Blueprint('nl', __name__, url_prefix='/nl')
+
+
+@bp.route('/<path:dcid>')
```

Comment:
what dcid is this? as far as i know, the landing page will have a search box where the user specifies a query/sentence. We then infer the SV dcid and place(s) dcid. So, shouldn't this load without the dcid? I see that it is being used to get place data ( place_page_data = dc.get_landing_page_data(dcid, 'Overview', []) ), but at this point we don't know what place, right?